src/pages/identityManagement/identityManagement.html

```diff
@@ -0,0 +1,38 @@
+<ion-header>
+  <ion-navbar color=primary>
+    <button ion-button menuToggle>
+      <ion-icon name="menu"></ion-icon>
+    </button>
+    <ion-title>Identity Management</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content padding>
+
+  <ion-grid style="height: 100%">
+    <ion-row justify-content-center align-items-center style="padding-top: 25%">
+      <img src="./assets/imgs/aerogear_logo_600px.svg" class="logo" height="100px" width="100%">
+    </ion-row>
+    <ion-row justify-content-center align-items-center>
```

Comment:
If we remove the second `row` and nest it as `grid -> row -> col` it will be a bit more responsive, at the moment the rows overlap when you flip the phone on its side.